subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -155,9 +155,18 @@ class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec {
         """
 
         when:
-        succeeds 'copy'
+        run 'copy'
         then:
+        executedAndNotSkipped ':copy'
         outputDirectory.list().contains input.name
+
+        when:
+        run 'copy'
```

Comment:
We should remove the broken symlink before we run the task again.
```
brokenLink.delete()
run 'copy'
```

Reply:
Done